examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -629,13 +631,18 @@ function vertical_diffusion_boundary_layer_tendency!(Yₜ, Y, p, t)
         one.(Fields.field_values(ᶠz_a)) # TODO: fix VIJFH copyto! to remove this
     @. ᶠK_E = eddy_diffusivity_coefficient(norm(ᶠv_a), ᶠz_a, ᶠinterp(ᶜp))
 
+    # TODO: Revisit z_surface construction when "space is not same instance" error is dealt with
+    z_field = Fields.coordinate_field(Y.c).z
+    ᶜz_interior = Fields.field_values(Fields.level(z_field, 1))
+    ᶠz_surface = Fields.field_values(z_sfc)
```

Comment:
```suggestion
    z_surface = Fields.field_values(z_sfc)
```
(I think interior and surface are clear enough that we don't need a subscript c or f, but up to you.)